src/main/java/Assignment5.java

```diff
@@ -0,0 +1,9 @@
+import java.util.Arrays;
+public class Assignment5 {
+    public static void main(String[] args) {
```

Comment:
Instead of writing this code in void main, lets define a function for this. and lets create test in src/test/* package which tests your function